network.go

```diff
@@ -174,6 +181,7 @@ func (n *network) MarshalJSON() ([]byte, error) {
 	netMap["endpointCnt"] = n.endpointCnt
 	netMap["enableIPv6"] = n.enableIPv6
 	netMap["generic"] = n.generic
+	netMap["persist"] = n.persist
```

Comment:
If we are skipping this object to be put in store why are we even trying to marshal it?